### PR TITLE
chore(react-progress): adopt custom JSX pragma

### PR DESCRIPTION
**Pull request by bsunderhus**
## New Behavior

1. Adopts `react-jsx-runtime` custom pragma on `react-progress`

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/Avatar-mount_base.html">714</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/Avatar-mount.html">696</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/Button-mount_base.html">381</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/Button-mount.html">381</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/Field-mount_base.html">1302</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/Field-mount.html">1298</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/FluentProvider-mount_base.html">899</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/FluentProvider-mount.html">895</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/FluentProviderWithTheme-mount_base.html">114</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/FluentProviderWithTheme-mount.html">118</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">92</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">87</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">102</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">114</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/InfoButton-mount_base.html">21</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/InfoButton-mount.html">15</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/MakeStyles-mount_base.html">1097</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/MakeStyles-mount.html">1126</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/Persona-mount_base.html">2033</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/Persona-mount.html">2037</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/SpinButton-mount_base.html">1590</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27617/perf-test-react-components/SpinButton-mount.html">1644</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                    Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `68.803 kB`<br />`18.728 kB` |   `68.69 kB`<br />`18.686 kB` | `-113 B` <br />`-42 B`  |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `208.156 kB`<br />`57.857 kB` | `208.043 kB`<br />`57.817 kB` | `-113 B` <br />`-40 B`  |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |   `38.886 kB`<br />`12.41 kB` |  `38.773 kB`<br />`12.375 kB` | `-113 B` <br />`-35 B`  |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressBar.fixture.js'>ProgressBar</abbr>                                                                        |   `16.409 kB`<br />`5.855 kB` |   `16.445 kB`<br />`5.865 kB` |    `36 B` <br />`10 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                     |       Size (minified/GZIP) |
| --------------------------------------------------------------------------------------------------------------------- | -------------------------: |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr> | `6.446 kB`<br />`2.185 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/4c57dc79fe7dfa0f6bede7741f90aad0d95ffacf'>4c57dc79fe7dfa0f6bede7741f90aad0d95ffacf</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: packages/react-components/react-progress/package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "@fluentui/react-shared-contexts": "^9.3.3",
     "@fluentui/react-theme": "^9.1.7",
     "@fluentui/react-utilities": "^9.8.0",
+    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
```
Comment (khmakoto) [inline packages/react-components/react-progress/package.json:39]: _nit:_ Can you move this above `@fluentui/react-shared-contexts` in line `36` to keep alphabetical order?

```suggestion
    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
    "@fluentui/react-shared-contexts": "^9.3.3",
    "@fluentui/react-theme": "^9.1.7",
    "@fluentui/react-utilities": "^9.8.0",
```

**Review by khmakoto — APPROVED** [review]

**Review by tomi-msft — APPROVED** [review]

**Pull request merged**